test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -101,17 +101,31 @@ def test_concept_set_container_enclave_to_db(self):
             # Teardown
             run_sql(con, f"DELETE FROM {table} WHERE concept_set_id = '{concept_set_id_succeed}';")
 
+#
+    # todo: fix: PKs (and other constraints) aren't set, so this inserts again and again without raising ane error.
+    #  background: originally thought the issue was this: https://github.com/jhu-bids/TermHub/issues/803, but it's #804
+    @unittest.skip("Skipping failing test for now. See: https://github.com/jhu-bids/TermHub/issues/804")
+    def test_cset_version_enclave_to_db__raises_err(self):  # aka test_code_sets_enclave_to_db()
+        """Test codeset_version_enclave_to_db() throws err when expected
+
+        Insert once so it exists in the table, then insert again.
+        """
+        table = 'code_sets'
+        with get_db_connection(schema=TEST_SCHEMA) as con:
+            codeset_id_fail = 1  # exists in the enclave & DB
+            cset_version_enclave_to_db(con, codeset_id_fail, [table], skip_if_already_exists=False)
+            self.assertRaises(
+                IntegrityError, cset_version_enclave_to_db, con, codeset_id_fail, [table], skip_if_already_exists=False)
+
     # TODO: test is failing. fix
-    @unittest.skip("Skipping failing/erroring test temporarily.")
+    # @unittest.skip("Skipping failing/erroring test temporarily.")
     def test_cset_version_enclave_to_db(self):  # aka test_code_sets_enclave_to_db()
```

Comment:
## `test_cset_version_enclave_to_db()`
TODO: WIP

Other changes:
- Split part of this into its own test, `test_cset_version_enclave_to_db__raises_err()`, which is currently being skipped due to #804.